wiegand/api.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/python3
+from RPi.GPIO import *
+import time
+import json
+
+class Wiegand26:
+    #Reads pin numbers for input pins from JSON
```

Comment:
make sure you follow the commenting convention. instead of `#sample comment`, this is better -> `# sample comment`.